lib/index.js

```diff
@@ -68,4 +68,7 @@ Elevio.prototype.identify = function(identify) {
   if (plan) user.groups = [plan];
   if (objectKeys(traits).length > 0) user.traits = traits;
   window._elev.user = user;
+  if (typeof window._elev.setUser !== 'undefined') {
```

Comment:
Can we change this to === to 'function' instead? Just to avoid any uncaughts in the browser if, for whatever reason, this property gets overwritten by some defined, non-function value. :)